src/algorithms.jl

```diff
@@ -167,6 +167,27 @@ struct VCABM5 <: OrdinaryDiffEqAdaptiveAlgorithm end
 
 struct VCABM <: OrdinaryDiffEqAdamsVarOrderVarStepAlgorithm end
 
+# IMEX Multistep methods
+
+struct ABCN2{CS,AD,F,FDT,K,T,T2} <: OrdinaryDiffEqAlgorithm
```

Comment:
This is a Newton algorithm. OrdinaryDiffEqImplicitAlgorithm

Reply:
or `OrdinaryDiffEqImplicitExplicitAlgorithm` ?

Reply:
no reason to categorize it that deep. At that point it wouldn't be as useful. This is to catch the implicit restart features.